source/making_payments/index.html.md.erb

```diff
@@ -5,68 +5,91 @@ weight: 70
+</style>
 
-You will receive the ``next_url``  to which you should direct the user to
-complete their payment.
+<br>
 
-## Tracking the progress of a payment
+<div class="govuk-warning-text">
+  <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+  <strong class="govuk-warning-text__text">
+    <span class="govuk-warning-text__assistive">Warning</span>
+You must not expose the URL with the `paymentId` or the `next_url` publicly, for example as a URL parameter or in an unencrypted cookie. You should store these values in encrypted cookies or in a database.
+  </strong>
+</div>
 
-You can track the progress of a payment while the user is on GOV.UK Pay using
-the <a
-href="https://gds-payments.gelato.io/api-explorer/gov-uk-pay-api/versions/1.0.2/v1/find-payment-by-id"
-target="blank">Find payment by ID</a> call [external link].
+## Track the progress of a payment
 
-NOTE: The status of the payment will go through several phases until it either
-succeeds or fails. See the [API reference
-section](/api_reference/#api-reference) for more details.
+You can track the progress of a payment while a user is on GOV.UK Pay using
+the <a href="https://gds-payments.gelato.io/api-explorer/gov-uk-pay-api/versions/1.0.2/v1/find-payment-by-id"
+target="blank">Find payment by ID</a> call [external link]
 
-## Choosing the return URL and matching user to payment
+The status of the payment will pass through several states until it either
+succeeds or fails. See the [“API reference”
+section](/api_reference/#api-reference) of this documentation for more
+details.
 
-For security reasons, GOV.UK Pay does not add the payment ID or outcome to
-your ``return_url`` as parameters.
+## Choose the return URL and match users to payments
 
-To match up a returning user with their payment, there are two recommended
-methods:
+For security reasons, GOV.UK Pay does not add the `paymentId` or outcome to
+your `return_url` as parameters.
 
-* use a secure cookie containing the Payment ID from GOV.UK Pay, issued by
-your service when the payment is created (before sending the user to
-``next_url``). Users will not be able to decrypt a secure cookie, so a
-fraudster could not alter the payment ID and intercept other users’ payments.
+You can match users with payments with an encrypted cookie, or by creating a
+secure random ID:
 
-* create a secure random ID (such as a UUID) and include this as part of the
-``return_url``, using a different ``return_url`` for each payment. Since a
-securely generated UUID is not guessable, fraudsters will not be able to
-intercept users’ payments.
+### Use an encrypted cookie
 
->Note: If you create an ID yourself, you’ll likely need to store this in a
-datastore mapped to the payment ID just after you create a payment.
+You can use a secure cookie containing the `paymentId` from GOV.UK Pay. Your
```

Comment:
This should say “encrypted cookie” rather than “secure cookie”.

Reply:
Indeed, changed